spotipy/model/playlist.py

```diff
@@ -30,7 +30,10 @@ class PlaylistTrackPaging(OffsetPaging):
     items: List[PlaylistTrack]
 
     def __post_init__(self):
-        self.items = [PlaylistTrack(**t) for t in self.items]
+        self.items = [
+            PlaylistTrack(**t) for t in self.items
+            if t['track'] is not None   # TODO: remove with Web API issue 958
```

Comment:
Should this None check be in PlaylistTrack instead?

Reply:
I don't think so. This filters the `None` tracks completely. The `PlaylistTrack` object doesn't carry much information without it.

Reply:
Does ignored PlaylistTracks affects counting position for playlist_tracks_add/playlist_tracks_reorder?

Reply:
As far as I know, there is no way of adding an empty track to a playlist. The Web API issue gives a reference playlist, but even it doesn't return a null track. So I wouldn't be able to test this in any way. The guy who also referenced this issue created one for a library for Kotlin. Its maintainer slapped a `wontfix` on it. I'm leaning towards the same treatment. But now that this change is in, we can wait for the issue to be closed some time and remove this accordingly.

Reply:
it is actually documented feature:
"Information about the tracks of the playlist. Note, a track object may be null. This can happen if a track is no longer available."
https://github.com/felix-hilden/spotipy/issues/45#issuecomment-552670834

Reply:
Hmm, okay, better to do it then. I've missed the notification on your comment apparently.